dice-game-ui-angular/src/app/services/game-storage-service.ts

```diff
@@ -18,7 +18,8 @@ export class SavedGameEngine {
 export abstract class GameStorageService {
   abstract restoreGame(guid: Guid): GameEngine;
   abstract removeGame(guid: Guid): void;
-  abstract getAllSavedGames(): SavedGameEngine[];
+  abstract getPlayerSavedGames(playerId: string): SavedGameEngine[];
+  protected abstract getAllSavedGames(): SavedGameEngine[];
```

Comment:
Protected methods should be located after all public methods.

Reply:
Got it